Poker.BE/Poker.BE.Domain/Game/Room.cs

```diff
@@ -139,13 +139,13 @@ public void EndCurrentHand()
         /// UC027 Choose Play Move
         /// </summary>
         /// <see cref="https://docs.google.com/document/d/1OTee6BGDWK2usL53jdoeBOI-1Jh8wyNejbQ0ZroUhcA/edit#heading=h.8f3okxza6g2d"/>
-        public void ChoosePlayMove(Round.Move move)
+        public void ChoosePlayMove(Round.Move move, int amountToBet)
```

Comment:
ok, what do you think of that:
we'll just return the totalRaise field that holds the highest raise so far at the table. that will let the next player know what's the amount of money he needs to invest in order to keep playing.
that's enough, right?